config.py

```diff
@@ -792,11 +792,16 @@
 to **not** specify rejection thresholds for EOG and ECG channels here –
 otherwise, ICA won't be able to "see" these artifacts.
 
+If `'auto'` (default), use `autoreject` to find suitable rejection thresholds
```

Comment:
can you put a ref to autoreject webpage?

Reply:
also an important detail is that you use here so-called autoreject global and not local.
This needs to be clarified in the doc.